test/Transforms/test-remapped-value.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: mlir-opt %s -test-remapped-value | FileCheck %s
+
+// Simple test that exercises ConvertPatternRewriter::getRemappedValue.
+func @remap_input_1_to_1(%arg0: i32) {
+  %0 = "test.one_variadic_out_one_variadic_in1"(%arg0) : (i32) -> i32
+  %1 = "test.one_variadic_out_one_variadic_in1"(%0) : (i32) -> i32
+  "test.return"() : () -> ()
+}
+// CHECK-LABEL: func @remap_input_1_to_1
+// CHECK-NEXT: "test.one_variadic_out_one_variadic_in1"(%arg0, %arg0)
```

Comment:
Can you avoid hard coding the SSA value names here? Can you also add '-legalize' after the 'test' part of the file name? Otherwise it isn't clear what this is testing.